### ISSUE TITLE
[Druid] Repurposed Lava Dredger is not granting AP to druids in feral forms.

### ISSUE THREAD

**Issue by annaqtie** [SallyWhitemane/Mograine#747]
Description: Repurposed Lava Dredger is not granting AP to druids in feral forms. This BoA was made solely for this reason.

Describe the bug you have encountered. Why do you think it's a bug? BoA Repurposed Lava Dredger is not granting Feral AP.

How to reproduce: Equip the BoA on a druid in cat/bear form.

Please explain what you did when you encountered the bug. What is your class, where were you when it occured. What can we do to reproduce the bug ourselves? I equipped the BoA on a druid. Equip a BoA on the Druid.

How it should work: It should grant AP on druid in bear/cat form.

Please explain how should it work. Describe the expected behavior and back it with evidence, such as YouTube video / screenshot / comments from relevant date (when WotLK was live on retail). 

TLDR; Doesn't give druids AP when it should.
![CB11A0F9-D5E5-4E16-AA13-483ADE7C7CD7](https://user-images.githubusercontent.com/78295842/107607055-be384e80-6c40-11eb-86ad-8e5b9b8bb700.png)

**Comment by alexspew** [comment]
Item works fine as it should. It starts granting more attack power in Cat,Bear,Dire Bear and Moonkin forms only when player has reached around level 58.
